docs/designers-developers/developers/backward-compatibility/README.md

```diff
@@ -1 +1,57 @@
 # Backward Compatibility
+
+Gutenberg follows the WordPress backwards compatibility principles. This means Backward compatibility for producation public API is guaranteed. In some rare occasions breaking backward compatibility is unavoidable, in that case, the breakage should be as small as possible and the breakage should be documented as clearly as possible to third-party developpers using devnotes.
+
+## What qualifies as a production public API:
```

Comment:
Do you mean "production-ready"?

Reply:
non just production. Some APIs are not production APIs

Reply:
Hm, should it be named production API then?

Reply:
you mean remove "public"?

Reply:
Oh, no :D I meant: if these are not just "production APIs", why call them a "production public API"?

Reply:
That section explains the difference between production and non production APIs and which ones qualifies as public producation APIs like stated in the sentence above.

Reply:
It would be clearer to me if this was phrases as "production ready public api"

Reply:
why "ready", if it's there it's ready?

Reply:
It's just common phrasing I've seen so it reads better _to me_.  In other words, a "public api ready for production vs a public api not ready for production" (which you clarify is distinguished by what package it is built to in this section). Just an observation, it's not a sticky point for me.

Reply:
But that's not the meaning here. I don't want to make any difference between API ready or not for production. I want to differentiate dev tools from production code.

Reply:
🤔 hmm... ya so I guess what you're distinguishing between here is packages built for production use vs packages built for development use and both have public apis but we only care about backwards-compatibility when it comes to packages built for production?

If so, then maybe the heading could be:

> What qualifies as a public api for production use?

Again though, this is a nit.  It's completely fine with me if you just want to resolve and use the text as is.

Reply:
I'm leaning towards keeping it as is because it's the same term used in the previous sentence. We can iterate of course.

Reply:
I don't fully get the term. You're saying that "production public API" consists of "production packages" and "development packages". This is a bit confusing to me. So the "development packages" is also "production public API"?

Reply:
production public API is just public APIs exported from production packages

Reply:
The confusion comes from the fact that we never address the question:

>"What qualifies as a production public API?".

Somewhere after defining the two types of packages we have to write:

"A production public API is a public API from a production package."